registry/verify-namespace-owner/verify_namespace_owner.go

```diff
@@ -99,6 +100,11 @@ func parseConfig(tk toolkit.Toolkit) (config, error) {
 		return config{}, toolkit.FailedError("namespace must be set")
 	}
 
+	c.ListNamespaces, ok = tk.GetInputList([]string{"list-of-namespaces"})
+	if !ok {
+		return config{}, toolkit.FailedError("namespace must be set")
```

Comment:
Instead of returning an error, I think we'll want to set a default list of blocked-namespaces. In fact, we will want to append those defaults to any list provided as input too.